### PR TITLE
Fixing broken get-started links

### LINKED ISSUE #11932: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/
I arrived on this page through: https://www.google.com/

### LINKED ISSUE #11938: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/
I arrived on this page through: https://www.google.com/

### LINKED ISSUE #11939: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/
I arrived on this page through: https://docs.docker.com/get-started/

### LINKED ISSUE #11948: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/
I arrived on this page through: android-app://com.google.android.gm/

### LINKED ISSUE #11950: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/
I arrived on this page through: https://www.google.com/

### LINKED ISSUE #11953: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/
I arrived on this page through: https://www.google.com/

### LINKED ISSUE #11955: 404 at /get-started/part3/
I found a broken link : https://docs.docker.com/get-started/part3/

This link was found in the welcome mail after verifying my email address.

image

### LINKED ISSUE #11957: Docker tutorial part two is gone
### Problem description

Last week, I started to follow the official getting started guide. This week, suddenly the page <https://docs.docker.com/get-started/part2/> was gone.

### Problem location

- I saw a problem on the following URL: <https://docs.docker.com/get-started/part2/>

### Suggestions for a fix

I’d expect at least a redirect (or information on why the tutorial is no longer published).

### LINKED ISSUE #11968: 404 at /get-started/part3/
I found a broken link : https://docs.docker.com/get-started/part3/

I received this email: [Docker Hub] Welcome to Docker!
I clicked on: Share your image on Docker Hub

### LINKED ISSUE #11969: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/
I arrived on this page through: https://docs.docker.com/

### LINKED ISSUE #11970: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/

### LINKED ISSUE #11981: 404 at /get-started/swarm-deploy/part2.md
I found a broken link : https://docs.docker.com/get-started/swarm-deploy/part2.md
I arrived on this page through: https://docs.docker.com/get-started/swarm-deploy/

### LINKED ISSUE #11987: 404 at /get-started/swarm-deploy/part2.md
I found a broken link : https://docs.docker.com/get-started/swarm-deploy/part2.md
I arrived on this page through: https://docs.docker.com/get-started/swarm-deploy/

### LINKED ISSUE #11997: 404 at /get-started/part3/
I found a broken link : https://docs.docker.com/get-started/part3/
I arrived on this page through: https://mail.qq.com/

### LINKED ISSUE #11998: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/
I arrived on this page through: https://mail.qq.com/

### LINKED ISSUE #12001: 404 at /get-started/kube-deploy/part2.md
I found a broken link : https://docs.docker.com/get-started/kube-deploy/part2.md
I arrived on this page through: https://docs.docker.com/get-started/kube-deploy/

### LINKED ISSUE #12017: 404 at /get-started/kube-deploy/part2.md
I found a broken link : https://docs.docker.com/get-started/kube-deploy/part2.md
I arrived on this page through: https://docs.docker.com/get-started/kube-deploy/

### LINKED ISSUE #12043: 404 at /get-started/part2/
I found a broken link : https://docs.docker.com/get-started/part2/
I arrived on this page through: https://docs.docker.com/

### PR DESCRIPTION
**Pull request by sudo-bmitch**
Signed-off-by: Brandon Mitchell <git@bmitch.net>

### Proposed changes

Add redirects for /get-started/part2/ and /get-started/part3/ to fix broken links from emails.

### Related issues (optional)

fixes https://github.com/docker/docker.github.io/issues/12043
closes https://github.com/docker/docker.github.io/pull/12003

This updates PR #11838 

fixes #11932 
fixes #11938 
fixes #11939 
fixes #11948 
fixes #11950 
fixes #11953 
fixes #11955 
fixes #11957
fixes #11968
fixes #11969
fixes #11970
fixes #11981
fixes #11987
fixes #11997
fixes #11998
fixes #12001
fixes #12017
fixes #12043

### CODE REVIEW

**Comment by sudo-bmitch** [comment]
Fix can be seen with the redirects from:

- https://deploy-preview-11960--docsdocker.netlify.app/get-started/part2/
- https://deploy-preview-11960--docsdocker.netlify.app/get-started/part3/

**Review by nebuk89 — APPROVED** [review]
Thanks for the fixes looks good to me

**Comment by sudo-bmitch** [comment]
Fixes #11987, fixes #11981.

File: get-started/kube-deploy.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: Learn how to describe and deploy a simple application on Kubernetes
 ## Prerequisites
 
 - Download and install Docker Desktop as described in [Orientation and setup](index.md).
-- Work through containerizing an application in [Part 2](part2.md).
+- Work through containerizing an application in [Part 2](02_our_app.md).
```
Comment (thaJeztah) [inline get-started/kube-deploy.md:10]: oh! just left a comment on https://github.com/docker/docker.github.io/pull/12003#issuecomment-754577462

was wondering if the new content is a "drop-in" replacement for the old one in this context 🤔

**Review by thaJeztah — APPROVED** [review]
LGTM

let's merge this one to fix the broken links; we can continue discussion if (some of) the old content should be restored

File: get-started/kube-deploy.md (hunk 1 above)
Comment (sudo-bmitch) [reply get-started/kube-deploy.md:10]: I think some of the content got split across multiple pages, so for the generic references, linking to the first page of the set made the most sense to me. This probably needs to be reviewed on a case-by-case basis.

**Pull request merged**